### PR TITLE
Wishlist and Cart Management pages created

### PR DESCRIPTION
**Pull request by mdarif6**
Wishlist and Cart Management pages created

### CODE REVIEW

**Review by Hnh013 — APPROVED** [review]
looks good to merge !

File: cart_page.css [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+  grid-row: 2/3;
+  background-color: var(--ha-light-background-color);
+}
+.cart-heading {
+  padding: 1rem;
+}
+
+.h-cart-container {
+  justify-content: center;
+
```
Comment (samyakshah3008) [inline cart_page.css:23]: You can remove this whitespace while refactoring

**Review by samyakshah3008 — APPROVED** [review]
Next time please attach the preview link as well or some screenshots of live server so as for better experience:)
Coding part is good, clean code, ready to merge

**Review by ravikumar1002 — APPROVED** [review]
looks good to merge

**Comment by mdarif6** [comment]
> Next time please attach the preview link as well or some screenshots of live server so as for better experience:) Coding part is good, clean code, ready to merge

Thanks, noted the same

**Pull request merged**